tests/integration/main.go

```diff
@@ -37,28 +37,12 @@ func main() {
 	dmc.ValidateStringLiterals()
 	dmc.startDocker()
 
-	mul := MultiConsumerTestRun()
```

Comment:
Instead of just deleting these could we have the option to run them manually, just not on CI

Multiple consumer tests seem useful and I wouldn't want that previous work to goto waste